manifests/params.pp

```diff
@@ -5,6 +5,7 @@
   $service_enable        = pick($mongodb::globals::service_enable, true)
   $service_ensure        = pick($mongodb::globals::service_ensure, 'running')
   $service_status        = $mongodb::globals::service_status
+  $restart               = false
```

Comment:
Previously the service was notified when the config file changed. Defaulting this to false makes it a backwards-incompatible change. Would it be terribad to default to true?